crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -385,9 +385,14 @@ def prepare_nodes_for_os_upgrade
     upgrade_nodes.each do |node|
       node["target_platform"] = admin_node["provisioner"]["default_os"]
       node["crowbar_wall"]["crowbar_upgrade_step"] = "prepare-os-upgrade"
+      # skip initial keystone bootstrapping
+      node["keystone"]["bootstrap"] = true if node["keystone"]
       node.save
     end
 
+    # unset `db_synced` flag for OpenStack components
+    ::Openstack::Upgrade.unset_db_synced
```

Comment:
I'm wondering if it's really the best location for this, as this will have an impact if the user cancels the upgrade. Isn't there a place where we can do that a bit later?